drivers/hmis/app/graphql/types/application/user.rb

```diff
@@ -99,5 +101,17 @@ def enrollment_access_summaries(filters: nil)
           project_ids: filters&.project,
         )
     end
+
+    def staff_assignments
+      # n+1 queries, needs optimization to be performant on collections
+      object.staff_assignments.
+        joins(:household).
```

Comment:
I did a little testing locally and it seems like this query is kind of slow. My modestly sized db with 400k enrollments loads this in about 2 seconds. Probably worth optimizing as it's going to be the landing page for most folks. Maybe it could be done in a follow up issue

Reply:
That's a good idea @ttoomey and I'm happy to do it as part of this ticket rather than a follow-up issue. Could we pair on this perhaps?